src/components/grid-aware/ThreeParagraphBlock/ThreeParagraphBlock.js

```diff
@@ -99,29 +159,37 @@ const ThreeParagraphBlock = ({
         <ParagraphBlock
           title={paragraph3.title}
           description={paragraph3.description}
+          button={paragraph3.button}
         />
       </div>
-      <div className={s.image3Wrapper}>
-        <img src={image3.url} alt={image3.alt} />
+      <div className={s.rightImageWrapper}>
+        <img className={s.image} src={rightImage.url} alt={rightImage.alt} />
       </div>
     </div>
   );
 
-  const GridAreaBottom = () => (
-    <div className={s.gridAreaBottom}>
-      <CTABlock title={ctaTitle} buttons={ctaButtons} />
-    </div>
-  );
+  let GridAreaBottom;
```

Comment:
I think we should reserve `CapitalCamelCased` names for things that act as classes (including React function components). The change you made here causes it to be a normal variable that happens to hold JSX content. I think you have two options:

1. Rename this to `gridAreaBottom`
2. Keep the definition as a React function component with no problem, and put the conditional inside of the body, kind of like this:

```jsx
const GridAreaBottom = () => {
  if (!ctaTitle || !ctaButtons) {
    return null;
  }
  return (
    <div>...</div>
  );
}

Reply:
I like #2, it looks cleaner.